source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,254 @@
+    - ``IncidentID``: Unique identifier of the incident where the item was.
+    - ``UserID``: Unique identifier of the user that removed the item."
+    "Checklist item renamed.", "- Any user edit the contents of a checklist item through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item was.
+    - ``UserID``: Unique identifier of the user that removed the item."
+    "Checklist item moved.", "- Any user moves the position of a checklist item in the list through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item is.
+    - ``UserID``: Unique identifier of the user that edited the item."
+    "Unchecked checklist item checked.", "- Any user checks an unchecked checklist item through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item is.
+    - ``UserID``: Unique identifier of the user that checked the item."
+    "Checked checklist item unchecked.", "- Any user unchecks a checked checklist item through the incident details view, in the RHS.", "
+    - ``IncidentID``: Unique identifier of the incident where the item is.
+    - ``UserID``: Unique identifier of the user that checked the item."
+     "Playbook created.", "- Any user clicks on the ``+ New Playbook`` button on the backstage and saves it.", "
+    - ``PlaybookID``: Unique identifier of the playbook.
+    - ``TeamID``: Unique identifier of the team where the playbook is created.
+    - ``NumChecklists``: Number of checklists this playbook has after the update.
+    - ``TotalChecklistItems``: Number of checklist items, among all checklists, this playbook has after the update."
+     "Playbook deleted.", "- Any user clicks on the ``Delete`` button next to a playbook on the Backstage and confirms.", "
+    - ``PlaybookID``: Unique identifier of the playbook.
+    - ``TeamID``: Unique identifier of the team where the playbook was located.
+    - ``NumChecklists``: Number of checklists this playbook had immediately prior to deletion.
+    - ``TotalChecklistItems``: Number of checklist items, among all checklists, this playbook had immediately prior to deletion."
+    
+    
+Glossary
+^^^^^^^^
+
+* **Incident**: An event requiring the coordinated actions of one or more Mattermost users. An incident is either ongoing or closed.
+* **Playbook**: A set of steps to execute as part of resolving an incident. It consists of one or more checklists, with each checklist item representing a single step.
+* **Commander**: The Mattermost user currently responsible for transitioning an incident from ongoing to closed.
+* **Incident channel**: A Mattermost channel dedicated to real-time conversation about the incident.
+* **Incident member**: A Mattermost user with access to the corresponding incident channel.
```

Comment:
Should this be "participant"?

Reply:
I think so. Looks like we are using "channel member" as usual but "incident participant".